src/Promitor.Integrations.AzureMonitor/RequestHandlers/AzureResourceManagerThrottlingRequestHandler.cs

```diff
@@ -46,11 +54,19 @@ public AzureResourceManagerThrottlingRequestHandler(string tenantId, string subs
             _metricSinkWriter = metricSinkWriter;
             _metricsCollector = metricsCollector;
 
+            string id = applicationId;
+
+            if (authenticationMode == AuthenticationMode.ManagedIdentity)
+            {
+                id = string.IsNullOrEmpty(managedIdentityId) ? "system-assigned-identity" : managedIdentityId;
+            }
+
             _metricLabels = new Dictionary<string, string>
             {
                 {"tenant_id", tenantId},
                 {"subscription_id", subscriptionId},
-                {"app_id", applicationId}
+                {"authentication", authenticationMode.ToString()},
```

Comment:
Not sure if this is valuable info since this is applicable for all metrics and implementation detail?

Reply:
Do you want to log only tenant_id and subscription_id (and remove auth) ?

Reply:
Yeah, maybe that makes more sense but that's a breaking change so would like to keep `app_id` for now